### ISSUE TITLE
RStudio version is very old

### ISSUE THREAD

**Issue by mbjones** [whole-tale/whole-tale#59]
Please provide a recent release of rocker.  

And can we figure out how to keep a newish version always available as updates are made to rocker?

**Comment by cboettig** [comment]
Would it be possible to pin to something like `rocker/geospatial:latest`?  (seems like some geospatial packages are already installed, but not sure what is being used as the base image.)

**Comment by Xarthisius** [comment]
@cboettig we can pin to a specific version. Pinning to a rolling tag makes reproducibility "slightly" harder...

**Comment by cboettig** [comment]
agree, pinning to a specific version would be ideal.  Using any of those should now also mean that the default CRAN repo will point to an MRAN snapshot with date appropriate to the version, which should also make for a more reproducible `install.packages()` behavior.  Currently it looks like `options(repos="https://cran.rstudio.com")` instead.
